### ISSUE TITLE
Docusaurus install 手順

### ISSUE THREAD

**Issue by yabuki** [npocccties/git-book-linked#1]
# はじめに

浅井さんに、お奨めしてもらった、Docsaurus に関して、手順および判断基準を残すために書く。

##  目的

Docsaurus が日本語環境でも使いやすいかの検証をし、使うかどうかの可否を決める。

# メモと手順

*  nvm.sh を利用しているので、npm i -g yarn を実行
```
npm i -g yarn

> yarn@1.22.10 preinstall /home/yabuki/.nvm/versions/node/v12.16.3/lib/node_modules/yarn
> :; (node ./preinstall.js > /dev/null 2>&1 || true)

/home/yabuki/.nvm/versions/node/v12.16.3/bin/yarn -> /home/yabuki/.nvm/versions/node/v12.16.3/lib/node_modules/yarn/bin/yarn.js
/home/yabuki/.nvm/versions/node/v12.16.3/bin/yarnpkg -> /home/yabuki/.nvm/versions/node/v12.16.3/lib/node_modules/yarn/bin/yarn.js
+ yarn@1.22.10
added 1 package in 2.798s
```
* ドキュメントでは、強力にdocsaurus@v2 を推しているので、まずはその通りに。
  * [Introduction | Docusaurus](https://v2.docusaurus.io/docs/)
  * [Installation | Docusaurus](https://v2.docusaurus.io/docs/installation)
  * requirement は満たした。
* git-book-linked のgh-pages を設定する。

**Comment by yabuki** [comment]
pluginsとして、 [remarkjs/remark: Markdown processor powered by plugins part of the @unifiedjs collective](https://github.com/remarkjs/remark) が提示されているが、これと、 [rehypejs/rehype: HTML processor powered by plugins part of the @unifiedjs collective](https://github.com/rehypejs/rehype) は何を指しているのか。

**Comment by yabuki** [comment]
[Site Search &amp; Discovery powered by AI | Algolia](https://www.algolia.com/)
が、docusaurus で使われいてる。これ検索につかえとの意味だったのか。確認がいる。無償枠があるのでよいという話なのかもしれない。

**Comment by yabuki** [comment]
[Markdown Features | Docusaurus](https://v2.docusaurus.io/docs/markdown-features#inline-table-of-contents)

```
import TOCInline from '@theme/TOCInline';

<TOCInline toc={toc} />;
```

をmdx.mdにコピペしても動かない。なんか間違えている。

で、yarn add @theme/TOCInline してもエラーになる。

**Comment by yabuki** [comment]
翻訳の機能

[Crowdin](https://crowdin.com/) が例として上がっているが、ディレクトリをこの通りに切って、ファイルを翻訳したら [Crowdin](https://crowdin.com/)  のサービスは不要なのか?

サービスを使った方がいいのは分かったが。

**Comment by dynamis** [comment]
> Site Search & Discovery powered by AI | Algolia
> が、docusaurus で使われている。これ検索につかえとの意味だったのか。確認がいる。無償枠があるのでよいという話なのかもしれない。

はい。インクリメンタル検索を全部自前でビルド時にインデックス作ってそれを GraphQL などで読み出す系のフレームワークやテーマもあったと思いますが、最近は無料枠のある Algolia を使うのが多いです。アカウント作って使う感じ、規模が小さいから無料枠で問題ないハズ。

> をmdx.mdにコピペしても動かない。なんか間違えている。

利用例 https://github.com/facebook/docusaurus/search?q=TOCInline&type=code 見る限りテーマ側で実装、export しているものを import しているようですが、サポートしているテーマで試されていますか？ (上のお試し手順を読めと言うことかもですが取り急ぎ)
